docs/testplan/Convergence measurement in data center networks.md

```diff
@@ -175,7 +177,8 @@ Effectively, on T1 routes are learned from 2 peers and forms ECMP.
 
 Fixed duration server traffic is generated for a bunch of destination prefixes, which will load balance by T1 towards its peers.<br/>
 During normal conditions, all the traffic should be received by tgen ports without experiencing any loss.<br/>
-Different triggers(planned/unplanned) mentioned above are performed at different layers (T1/T2/T3) and total packets lost are measured. 
+Different triggers(planned/unplanned) mentioned above are performed at different layers (T1/T2/T3) and total packets lost are measured.<br/> 
+For Keysight automated cases, we will be measuring convergence using "Packet Loss Duration (PLD)" stat, which will be default convergence measurement. To measure the flow which is having high convergence value, flow tracking will be used. Note, this will be limited by the tracking limit depending on hardware which is being used.
```

Comment:
Is `"Packet Loss Duration (PLD)"` an available counter in IXIA or it will be calculated by the tests? To use the flow tracking feature Will the number of prefixes used in the test parameterized?

Reply:
It is an available state in IxNetwork. Yes, the number of routes will be parameterized. If the number of routes is supported, it will calculate the prefix which is having the maximum loss. If not, it will calculate the average packet loss duration for whole prefixes.